### DISCUSSION TITLE
Mimir and pre-defined Alertmanager Config & Rules

### DISCUSSION THREAD

**Discussion by KoffeinKaio** [grafana/mimir#8672] [Help and support]
Hey,

im currently evaluation a deployment of mimir (still on the small side) and it works so far.
The biggest problem aside from the Documentation is getting Alertmanager config and Rules to work/to deploy with the Helm chart.

As far as I know Alertmanager loads a "fallbackConfig" if no config is provided for the tennant. As we only have one tennant (anonymous/0?) we have a default config.  But Alertmanager doesnt load it and says theres already a config available.
Which I cant delete with mimirtool alertmanager delete - so I cant get it to load.

Next thing is - how do I load the rules? Preferably I would like to load a rules.yaml or define them in the Helm chart somehow.

My goal is to use a setup of Grafana/Mimir/Alloy/Loki without the need for prometheus.

Is there an example available? A guide? Documentation?

**Comment by dimitarvdimitrov** [comment]
> Next thing is - how do I load the rules? Preferably I would like to load a rules.yaml or define them in the Helm chart somehow.



this is currently not possible with the Mimir-distributed helm chart: see https://github.com/grafana/mimir/issues/2861

But take a look at Grafana Alloy - that can provision mimir rules based on a terraform-like language
